tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -42,6 +42,27 @@ protected function setUp(): void
         ;
     }
 
+    public function testHasCoreServicesAlias(): void
+    {
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Admin\Pool');
```

Comment:
Please, use `::class` constants instead of hardcoded values. By instance:
```php
$this->assertContainerBuilderHasService(AdminPoolLoader::class);
```

Reply:
Done